cluster-autoscaler/cloudprovider/alicloud/alibaba-cloud-sdk-go/services/ess/describe_scaling_groups.go

```diff
@@ -167,6 +167,7 @@ type ScalingGroup struct {
 	RemovingCapacity             int             `json:"RemovingCapacity" xml:"RemovingCapacity"`
 	ScalingGroupName             string          `json:"ScalingGroupName" xml:"ScalingGroupName"`
 	ActiveCapacity               int             `json:"ActiveCapacity" xml:"ActiveCapacity"`
+	DesiredCapacity              int             `json:"DesiredCapacity" xml:"DesiredCapacity"`
```

Comment:
```suggestion
	DesiredCapacity              int             `json:"DesiredCapacity" xml:"DesiredCapacity"`
	EnableDesiredCapacity        bool            `json:"EnableDesiredCapacity,omitempty" xml:"EnableDesiredCapacity,omitempty"`
```

`EnableEnableDesiredCapacity` required. 
https://next.api.aliyun.com/document/Ess/2014-08-28/DescribeScalingGroups